lib/utils/displayAmountInEther.js

```diff
@@ -1,6 +1,7 @@
 import { ethers } from 'ethers'
 import { numberWithCommas } from 'lib/utils/numberWithCommas'
 
+// TODO: How does this fit in with flagship formatters
```

Comment:
I believe my goal recently was to remove `displayAmountInEther()` and simply use `numberWithCommas()` everywhere instead. Looks like there may be only 1 place where we use `displayAmount...` in the ref app, and can prob delete this and replace that 1 instance with `numberWithCom...`